tests/ert_tests/ert3/config/test_stages_config.py

```diff
@@ -230,12 +235,12 @@ def test_stage_unknown_field(base_stage_config):
 
 def test_stage_input_immutable(base_stage_config):
     config = ert3.config.load_stages_config(base_stage_config)
-
+    first_input_record = base_stage_config[0]["input"][0]["record"]
```

Comment:
I like the verbosity; the only downside is that `first_input_record` makes you think it's the actual record while it refers only to its name. The same applies for other `first_input_record` occurrences.

Reply:
Thanks! `first_input_record_name` should fix this?

Reply:
or just `input_record_name`...